src/platform_impl/linux/x11/dnd.rs

```diff
@@ -45,20 +46,33 @@ pub struct Dnd {
     pub type_list: Option<Vec<xproto::Atom>>,
     // Populated by XdndPosition event handler
     pub source_window: Option<xproto::Window>,
+    // Populated by XdndPosition event handler
+    pub position: PhysicalPosition<f64>,
     // Populated by SelectionNotify event handler (triggered by XdndPosition event handler)
     pub result: Option<Result<Vec<PathBuf>, DndDataParseError>>,
+    // Populated by SelectionNotify event handler (triggered by XdndPosition event handler)
+    pub has_entered: bool,
```

Comment:
I think it's more like `dragging` or something like that? It's generally whether we have an on-going drag or not. Applies to the rest pretty much. Though, I don't mind entered that much, we can keep it as is.

Reply:
Renamed to `dragging`.